site/app/config/packages/old_sound_rabbit_mq.yml

```diff
@@ -30,3 +33,13 @@ old_sound_rabbit_mq:
             exchange_options: {name: 'update-project-info', type: fanout}
             queue_options:    {name: 'update-github-metadata'}
             callback:         update_github_metadata_consumer
+        web_notification:
+            connection:       default
+            exchange_options: {name: 'send-notification', type: fanout}
+            queue_options:    {name: 'send-notification'}
```

Comment:
The message currently goes through the following areas
- An Exchange first
- Queue next
- Both the consumers where `shouldHandle()` checks if it will be handled by the one it is in

But I guess we want a structure where it goes through
- An Exchange first
- The Queue of the specific Consumer
- To the specific Consumer

Reply:
I'll be working on this today itself

Reply:
So, I have come up with two promising architecture designs. I believe we need very small attention to detail here. I found there is another way here(it may be unnecessary).

# A Direct Exchange architecture
![direct](https://user-images.githubusercontent.com/26324376/60611672-81f60880-9de4-11e9-83ac-20f2117adf3c.jpg)

**Advantages**
- Less message load in the buffer

# A fanout Exchange architecture
![Fan](https://user-images.githubusercontent.com/26324376/60611917-1e200f80-9de5-11e9-8550-9532fafcf89b.jpg)

**Advantages**
- No routing key required as it broadcasts messages to both queues, simpler
- No logic required beforehand to generate a routing key

Reply:
We want to go for the "fanout" architecture as inserting a message should be fast (it's in the synchronous page rendering path in many cases).